typescript/cloudwatch/evidently-client-side-evaluation-lambda/lambda-handler.py

```diff
@@ -0,0 +1,14 @@
+import boto3
+import botocore.config
+
+aws_config = botocore.config.Config(inject_host_prefix=False)
+
+
+def main(event, context):
+    client = boto3.client('evidently', endpoint_url='http://localhost:2772', config=aws_config)
```

Comment:
How'd this happen? Where's this come from?

Reply:
will add docs

Reply:
The nit that I have left here is that it's not *entirely* clear that by adding the special layer, there's a new network service added when the lambda is brought up for execution.